spec/cli/commands/list_spec.rb

```diff
@@ -4,4 +4,29 @@
 
 describe Ronin::Wordlists::CLI::Commands::List do
   include_examples "man_page"
+
+  describe '#run' do
+    context 'with given name' do
+      let(:name)      { 'list' }
+      let(:wordlists) { ['list1', 'list2'] }
+
+      it 'must list wordlists matching the specified name' do
+        expect(subject.wordlist_dir).to receive(:list).with(name).and_return(wordlists)
+        expect {
+          subject.run(name)
+        }.to output("  list1\n  list2\n").to_stdout
+      end
+    end
+
+    context 'witout given name' do
+      let(:wordlists) { ['wordlist1', 'wordlist2', 'wordlist3'] }
+
+      it 'must list all wordlists' do
+        expect(subject.wordlist_dir).to receive(:list).with(no_args).and_return(wordlists)
+        expect {
+          subject.run
+        }.to output("  wordlist1\n  wordlist2\n  wordlist3\n").to_stdout
```

Comment:
To make the specs more cross-platform, just in case someone runs them on Windows (which uses `\r\n` instead of `\n`), I tend to write multi-line Strings as either a heredoc or `[ ... ].join($/)`.

```ruby
.to output(
  <<~OUTPUT
      wordlist1
      wordlist2
      wordlist3
  OUTPUT
)
```

```ruby
.to output(
  [
    "  wordlist1",
    "  wordlist2",
    "  wordlist3",
    ""
  ].join($/)
)
```